ui/helpers/constants/design-system.ts

```diff
@@ -1,236 +1,227 @@
-  H7: 'h7',
-  H8: 'h8',
-  H9: 'h9',
-  Paragraph: 'p',
-  Span: 'span',
-};
-
-export const TEXT = {
-  DISPLAY_MD: 'display-md',
-  HEADING_LG: 'heading-lg',
-  HEADING_MD: 'heading-md',
-  HEADING_SM: 'heading-sm',
-  BODY_LG_MEDIUM: 'body-lg-medium',
-  BODY_MD: 'body-md',
-  BODY_MD_BOLD: 'body-md-bold',
-  BODY_SM: 'body-sm',
-  BODY_SM_BOLD: 'body-sm-bold',
-  BODY_XS: 'body-xs',
-  INHERIT: 'inherit',
-};
+export enum IconColor {
+  iconDefault = 'icon-default',
+  iconAlternative = 'icon-alternative',
+  iconMuted = 'icon-muted',
+  overlayInverse = 'overlay-inverse',
+  primaryDefault = 'primary-default',
+  primaryInverse = 'primary-inverse',
+  errorDefault = 'error-default',
+  errorInverse = 'error-inverse',
+  successDefault = 'success-default',
+  successInverse = 'success-inverse',
+  warningDefault = 'warning-default',
+  warningInverse = 'warning-inverse',
+  infoDefault = 'info-default',
+  infoInverse = 'info-inverse',
+  inherit = 'inherit',
+  goerli = 'goerli',
+  sepolia = 'sepolia',
+  goerliInverse = 'goerli-inverse',
+  sepoliaInverse = 'sepolia-inverse',
+}
 
-const NONE = 'none';
+export enum TypographyVariant {
+  H1 = 'h1',
+  H2 = 'h2',
+  H3 = 'h3',
+  H4 = 'h4',
+  H5 = 'h5',
+  H6 = 'h6',
+  H7 = 'h7',
+  H8 = 'h8',
+  H9 = 'h9',
+  paragraph = 'p',
+  span = 'span',
+}
 
-export const SIZES = {
-  XXS: 'xxs',
-  XS: 'xs',
-  SM: 'sm',
-  MD: 'md',
-  LG: 'lg',
-  XL: 'xl',
-  INHERIT: 'inherit', // Used for Text, Icon, and Button components to inherit the parent elements font-size
-  AUTO: 'auto',
-  NONE,
-};
+export enum TextType {
+  displayMd = 'display-md',
+  headingLg = 'heading-lg',
+  headingMd = 'heading-md',
+  headingSm = 'heading-sm',
+  bodyLgMedium = 'body-lg-medium',
+  bodyMd = 'body-md',
+  bodyMdBold = 'body-md-bold',
+  bodySm = 'body-sm',
+  bodySmBold = 'body-sm-bold',
+  bodyXs = 'body-xs',
+  inherit = 'inherit',
+}
 
-export const BORDER_STYLE = {
-  DASHED: 'dashed',
-  SOLID: 'solid',
-  DOTTED: 'dotted',
-  DOUBLE: 'double',
-  NONE,
-};
+export enum Size {
+  XXS = 'xxs',
+  XS = 'xs',
+  SM = 'sm',
+  MD = 'md',
+  LG = 'lg',
+  XL = 'xl',
+  inherit = 'inherit', // Used for Text, Icon, and Button components to inherit the parent elements font-size
+  auto = 'auto',
+  none = 'none',
+}
 
-export const BORDER_RADIUS = {
-  XS: SIZES.XS,
-  SM: SIZES.SM,
-  MD: SIZES.MD,
-  LG: SIZES.LG,
-  XL: SIZES.XL,
-  NONE,
-  PILL: 'pill',
-  FULL: 'full',
-};
+export enum BorderStyle {
+  dashed = 'dashed',
+  solid = 'solid',
+  dotted = 'dotted',
+  double = 'double',
+  none = 'none',
+}
 
-const FLEX_END = 'flex-end';
-const FLEX_START = 'flex-start';
-const CENTER = 'center';
+export enum BorderRadius {
+  XS = 'xs',
+  SM = 'sm',
+  MD = 'md',
+  LG = 'lg',
+  XL = 'xl',
+  none = 'none',
+  pill = 'pill',
+  full = 'full',
+}
 
-export const ALIGN_ITEMS = {
-  FLEX_START,
-  FLEX_END,
-  CENTER,
-  BASELINE: 'baseline',
-  STRETCH: 'stretch',
-};
+export enum AlignItems {
```

Comment:
Add a comment above this to explain our stance on this pluralized enum, considering that align-items is the css prop and that this will align *all* items not one.